google-cloud-firestore/lib/google/cloud/firestore/database.rb

```diff
@@ -0,0 +1,990 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google/cloud/firestore/collection"
+require "google/cloud/firestore/document"
+require "google/cloud/firestore/query"
+require "google/cloud/firestore/batch"
+require "google/cloud/firestore/read_only_transaction"
+require "google/cloud/firestore/transaction"
+
+module Google
+  module Cloud
+    module Firestore
+      ##
+      # # Database
+      #
+      class Database
+        ##
+        # @private The Service object.
+        attr_accessor :service
+
+        ##
+        # @private Creates a new Firestore Database instance.
+        def initialize service
+          @service = service
+        end
+
+        ##
+        # The project resource the Cloud Firestore batch belongs to.
```

Comment:
Does "batch" belong here? (1 of 3 usages in this file.)

Reply:
Fixed.